src/js/angular/ttyg/directives/chat-panel.directive.js

```diff
@@ -243,11 +256,13 @@ function ChatPanelComponent(toastr, $translate, TTYGContextService) {
             };
 
             const focusQuestionInput = () => {
-                const els = element.find('.question-input');
-                if (els.length) {
-                    els[0].focus();
-                }
-            };
+                $timeout(function() {
```

Comment:
Always document why a timeout was used, because often it's used for some quirky reason or a bad design! And I'm quite certain that this is the case in this situation. Did you explore what are the other ways to synchronize the question field focusing with the other mechanics that happens simulaneously?